app/utils/report/templates/test.txt

```diff
@@ -15,10 +15,16 @@ Test results summary
 {%- endfor %}
 
 
-Test failures
--------------
+{%- set print_header = true %}
 {%- for t in test_groups|sort(attribute='name') %} {# test_groups #}
   {%- if t.total["FAIL"] or t.regressions %} {# fail or regressions #}
+    {%- set failures = true %}
```

Comment:
Looks like `failures` is not used anywhere, is it just some leftover that should be removed?

Reply:
This is indeed a leftover after my experiments. I removed it.

Reply:
Thanks.